drivers/gl_context/glad/glad.h

```diff
@@ -150,12 +150,22 @@ typedef unsigned int GLhandleARB;
 typedef unsigned short GLhalfARB;
 typedef unsigned short GLhalf;
 typedef GLint GLfixed;
+#if defined(__APPLE__)
```

Comment:
The issue is fixed now. https://github.com/Dav1dde/glad/issues/70
Should regenerate glad files from the new updated version.
Do we still need this pull request, or is the other one working now?